vnet/spec/vnet/endpoints/1.0/mac_lease_spec.rb

```diff
@@ -41,7 +41,6 @@ def app
 
     accepted_params = {
       :interface_uuid => "if-test2",
-      :mac_address => "00:21:cc:da:e9:dc"
```

Comment:
`PUT mac_leases` still accepts a `:mac_address` parameter so it shouldn't be moved from the test.